PROTOCOL.md

```diff
@@ -63,10 +63,10 @@ Settings are applied using the following priority, highest first:
 
 ### Global Settings
 
-| Setting | Description                                          | Default |
-| ------- | ---------------------------------------------------- | ------- |
-| gpgPath | Optional path to gpg binary                          | `null`  |
-| stores  | List of password stores with store-specific settings | `null`  |
+| Setting | Description                                          | Default           |
+| ------- | ---------------------------------------------------- | ----------------- |
+| gpgPath | Optional path to gpg binary                          | `null`            |
+| stores  | List of password stores with store-specific settings | `{"default": {}}` |
```

Comment:
Since `configure` now returns a `defaultPath` back to the extension, maybe let the extension send it in here? So in the host app the default path is queries once, during the `configure` call.

Reply:
This is the default prior to the `configure` call being made. After that has been made, then the default path is included in the settings object.